### ISSUE TITLE
Phi discrepancies in HTML 

### ISSUE THREAD

**Issue by danscime** [JacquesCarette/Drasil#1809]
Working on SSP Mathjax (related to #479).
One problem I've noticed is that phi displays differently in HTML based on different environments.

First noticed in SSP TMods:
<img width="568" alt="Screen Shot 2019-08-07 at 1 45 34 PM" src="https://user-images.githubusercontent.com/28261339/62645176-ada36b80-b919-11e9-9234-30e6911ebdbb.png">

Characters taken from Language.Drasil.ShortHands.hs:
<img width="160" alt="Screen Shot 2019-08-07 at 1 38 50 PM" src="https://user-images.githubusercontent.com/28261339/62645488-5225ad80-b91a-11e9-969a-7c4b1f06b2a7.png">

Rendering plain text vs math in HTML:
<img width="231" alt="Screen Shot 2019-08-07 at 1 49 44 PM" src="https://user-images.githubusercontent.com/28261339/62645517-623d8d00-b91a-11e9-9b65-57438b94f6f3.png">

Produces:
<img width="200" alt="Screen Shot 2019-08-07 at 1 49 29 PM" src="https://user-images.githubusercontent.com/28261339/62645548-6f5a7c00-b91a-11e9-8cca-1b8fdbfad58b.png">

**Comment by smiths** [comment]
Good catch @danscime.  These kind of inconsistencies are something that we don't want to have in the long run.  I know I personally find them irritating when I read someone's document.  However, if the fix isn't easy, I think we can worry about this in the future.

**Comment by JacquesCarette** [comment]
In latex, there is $\phi$ and $\varphi$, which are different.  See [this math.stackexchange conversation](https://math.stackexchange.com/questions/1411557/notation-varphi-and-phi); the [wikipedia page](https://en.wikipedia.org/wiki/Phi) also has quite the conversation about it.

That the HTML somehow flips these around is a bug for sure. [and we should probably avoid using one of them anyways...].  I would expect this to be fairly easy to fix?

**Comment by muhammadaliog3** [comment]
Here is what the HTML looks like in now:
![tphi_table_drasil](https://user-images.githubusercontent.com/64814624/82692916-8dd1ff80-9c2e-11ea-9863-11ca608e84b8.png)
What the Language.Drasil.ShortHands.hs reads today
```
lPhi = Variable "ϕ"
vPhi = Variable "φ"
cPhi = Variable "Φ"
```
While the HTML for the phi in that table reads: 
```html
 <li>
         <em>φ&prime;</em> is the effective angle of friction (&deg;)
</li>
```
However ϕ in HTML translates to φ in HTML. In fact while I was typing this comment I found that by typing φ in the html block it translated to ϕ in the comment preview and that translated to φ in the SSP HTML example, but that is besides the point.

As you can see the issue no longer appears to be present. I could not find any phi discrepencies among the tables in the html file of the SSP. So I believe that this issue is no longer present.

**Comment by JacquesCarette** [comment]
Things do still seem to be a tad odd though.  The HTML you should above has ϕ displayed but it renders as φ (I had to cut-and-paste the opposite to get it to display properly in this comment!!!). So the end result seems ok.

I think the problem is, if we were to choose to use MathJax to display single-variable names, this problem would re-appear, as it would switch back, which is what the very last lines of Dan's report is about.  Did you test MathJax?

**Comment by muhammadaliog3** [comment]
Is MathJax just Latex with $$ or $ delimiters? If so here is the experiment when compiled
Here is experiment I used:
Code: 
```latex
\documentclass[12pt]{article}
\usepackage{unicode-math}
\begin{document}
\title BigExperiment
$${ ϕ }.$$
$φ$
$Φ$
\end{document}
```

The results when compiled as PDF: 
ϕ ------->𝜙
φ ------->𝜑
Φ ------->Φ

**Comment by JacquesCarette** [comment]
Yes, MathJax is just embedded LaTeX, delimited by combinations of `$`.

So your experiments show that things do swap around. So very weird!

**Comment by balacij** [comment]
I think this issue has to do with:
1. we use KaTeX (MathJaX) for 'equations' in the HTML version, while
2. we use unicode and HTML symbol natives to describe symbols outside of the KaTeX blocks in the HTML version.

I'm not sure if MathJax would be happy with us replacing all symbols with KaTeX symbols (due to resource usage), but it's worth exploring going down that route to assure consistency in symbol rendering. This would be the 'easiest' solution.
